app/controllers/sessions_controller.rb

```diff
@@ -7,16 +7,16 @@ def new
 
   def create
     if @user = login(session_params[:email], session_params[:password])
-      redirect_back_or_to root_url, notice: "Добро пожаловать!"
+      redirect_back_or_to root_url, notice: t('views.session.flash_messages.user_successfully_login')
     else
-      flash.now.alert = 'Неверный email или пароль'
+      flash.now.alert = t('views.session.flash_messages.wrong_email_or_password')
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.